Formula/j/jbigkit.rb

```diff
@@ -48,7 +48,7 @@ def install
   end
 
   test do
-    system "#{bin}/jbgtopbm #{pkgshare}/examples/ccitt7.jbg | #{bin}/pbmtojbg - testoutput.jbg"
+    system bin/"jbgtopbm #{pkgshare}/examples/ccitt7.jbg | #{bin}/pbmtojbg - testoutput.jbg"
```

Comment:
Nope, this should stay as it was because it's a string with spaces.

Reply:
yes!